ocdata/analysis.py

```diff
@@ -0,0 +1,66 @@
+"""
+This submodule contains the scripts for VASP calculations
+post-analysis:
+
+(1) Bader Charges Analysis
+(2) pDOS Analysis (TODO)
+"""
+
+__author__ = 'Javier Heras-Domingo'
+__email__ = 'jherasdo@andrew.cmu.edu'
+
+import os
+import sys
+import pickle
+from shutil import which
+
+import warnings
+warnings.filterwarnings("ignore")
+
+
+
+#Run Bader Analysis after vasp calculation
+def run_bader(outdir='./'):
+	"""
+	Arguments
+	--------------
+	outdir [str]: Directory of the vasp job
+
+	It returns bader.pkl file with bader analysis as a dict, 
+	where charge_transfer should be the interesting feature
+	per atom.
+
+	Example
+	--------------
+	from ocdata.analysis import run_bader
+
+	run_bader(outdir='./job_path')
+	"""
+	
+	#Add bader exec into the path
+	os.environ['PATH'] += os.pathsep + os.path.abspath('./ocdata')
```

Comment:
This only works if you are inside the `Open-Catalyst-Dataset` folder. We need to figure out a way to do this that'll work for anybody in any folder.

It also feels weird to modify the enviromental `$PATH` variable inside a Python script. Doing so may interfere with users' other scripts. And what if we call this script multiple times? Does it keep re-inserting the same path over and over again?

Is there any other way we can do this?